src/api/goods.ts

```diff
@@ -10,10 +10,14 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods.sort(
+      (good1, good2) => good1.name.localeCompare(good2.name),
+    ).filter(
+      (_good, index) => index < 5,
+    ));
```

Comment:
Great solution 👍 

But better use `slice` here, this way code would be more clear.